docs/benchmarks.rst

```diff
@@ -0,0 +1,68 @@
+
+.. inclusion-marker-start-do-not-remove
+
+
+Benchmarks
+==========
+
+
+.. image:: https://user-images.githubusercontent.com/16640218/38965607-bf5c46ca-4332-11e8-895a-b9c137e86013.png
+   :alt: 512-GPU Benchmark
+
+
+The above benchmark was done on 128 servers with 4 Pascal GPUs each connected by a RoCE-capable 25 Gbit/s network. Horovod
+achieves 90% scaling efficiency for both Inception V3 and ResNet-101, and 68% scaling efficiency for VGG-16.
+
+To reproduce the benchmarks:
+
+1. Install Horovod using the instructions provided on the `Horovod on GPU <https://github.com/horovod/horovod/blob/master/docs/gpus.rst>`__ page.
+
+2. Clone `https://github.com/tensorflow/benchmarks <https://github.com/tensorflow/benchmarks>`__
+
+.. code-block:: bash
+
+    $ git clone https://github.com/tensorflow/benchmarks
+    $ cd benchmarks
+
+
+3. Run the benchmark. Examples below are for Open MPI.
+
+.. code-block:: bash
+
+    $ horovodrun -np 16 -H server1:4,server2:4,server3:4,server4:4 \
+        python scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py \
+            --model resnet101 \
+            --batch_size 64 \
+            --variable_update horovod
+
+
+4. At the end of the run, you will see the number of images processed per second:
+
+.. code-block:: bash
+
+    total images/sec: 1656.82
+
+
+**Real data benchmarks**
```

Comment:
I just noticed that all these bold titles don't produce links that could be referenced.  Can you replace them with actual subsections (`~~~`)?

Reply:
Sure. I suppose you want me to create a new Pull Request for this?

Reply:
Yes, please :-)